python-inocybe-openswitch/models/base-etl-common.yang

```diff
@@ -0,0 +1,108 @@
+module base-etl-common {
+
+    namespace "http://www.dellemc.com/networking/os10/dell-base-etl-common";
+
+    prefix base-etl-common;
+
+    organization
+        "Dell EMC";
+
+    contact
+        "http://www.dell.com/support";
+
+    description
+        "This module contains a collection of common YANG definitions for ETL logging and event notification objects.";
+
```

Comment:
Since this is a new file, wondering if you need to get rid of all the previous references, and just put on with the latest date and the description as "Initial Version".  Upto you.

But in any case, you would need to put the latest revision.

Also, "Dell proprietary" is changed with "Network Platform Abstraction". This is done internally, and it would reflected in the opx-base-model in the next release